cvat-ui/src/components/signing-common/styles.scss

```diff
@@ -0,0 +1,296 @@
+    background: $background-color-1;
+    padding: $grid-unit-size * 6;
+    min-height: 620px;
+
+    .ant-form-item {
+        margin-bottom: $grid-unit-size*5;
+    }
+}
+
+.cvat-signing-form {
+    margin-top: $grid-unit-size*10;
+    margin-bottom: $grid-unit-size*16;
+    transition: $base-transition;
+}
+
+.cvat-signing-form-empty-socials {
+    margin-bottom: $grid-unit-size*28;
+}
+
+.cvat-signing-form-extended {
+    margin-top: $grid-unit-size*6;
+    margin-bottom: $grid-unit-size*14;
+}
+
+.cvat-password-reset-form-wrapper {
+    @extend .cvat-signing-form-wrapper;
+
+    h2 {
+        margin-bottom: 0;
+    }
+
+    .cvat-password-reset-tip {
+        font-weight: bold;
+        margin-bottom: $grid-unit-size*18;
+    }
+}
+
+.cvat-password-reset-form {
+    .cvat-credentials-form-item {
+        margin-bottom: $grid-unit-size*3;
+    }
+
+    margin-top: $grid-unit-size*6;
+    transition: $base-transition;
+}
+
+.cvat-register-form-wrapper {
+    @extend .cvat-signing-form-wrapper;
+
+    .cvat-credentials-form-item {
+        height: $grid-unit-size*9.5;
+
+        .ant-form-item-explain {
+            margin-bottom: $grid-unit-size*0.5;
+        }
+    }
+
+    .cvat-agreements-form-item {
+        height: $grid-unit-size*5.5;
+
+        .ant-form-item-explain {
+            margin-top: -$grid-unit-size;
+        }
+
+        .ant-checkbox-wrapper {
+            display: flex;
+            align-items: center;
+        }
+
+        .ant-checkbox {
+            top: 0;
+
+            .ant-checkbox-inner {
+                width: $grid-unit-size*4;
+                height: $grid-unit-size*2.25;
+                border-radius: 8px;
+                background: #d9d9d9;
+                border: none;
+
+                &:hover {
+                    border: none;
+                }
+
+                &::after {
+                    position: absolute;
+                    content: "";
+                    height: $grid-unit-size*1.5;
+                    width: $grid-unit-size*1.5;
+                    border-radius: $grid-unit-size;
+                    top: $grid-unit-size*0.4;
+                    left: $grid-unit-size*0.4;
+                    background-color: white;
+                    opacity: 1;
+                    transform: none;
+                }
+            }
+        }
+
+        .ant-checkbox-checked {
+            .ant-checkbox-inner {
+                width: $grid-unit-size*4;
+                height: $grid-unit-size*2.25;
+                border-radius: 8px;
+                background: $slider-color;
+                border: none;
+
+                &:hover {
+                    border: none;
+                }
+
+                &::after {
+                    position: absolute;
+                    content: "";
+                    height: $grid-unit-size*1.5;
+                    width: $grid-unit-size*1.5;
+                    border-radius: $grid-unit-size;
+                    top: $grid-unit-size*0.4;
```

Comment:
We should use ``grid-unit-size`` only with integer values. It was the core idea of introduction it.

Reply:
In this case we have `toggle` which is quite small to `$grid-unit-size` so its hard to use just integers. I tried to make it a bit bigger so there is less floats here

Reply:
What I mean here that `$grid-unit-size: 8px` is quite large for such thing like toogle. So in that case using something like `$grid-unit-size * 0.5` should be fine